packages/flutter/lib/src/material/back_button.dart

```
@@ -84,7 +87,10 @@ class BackButton extends StatelessWidget {
       color: color,
       tooltip: 'Back', // TODO(ianh): Figure out how to localize this string
       onPressed: () {
-        Navigator.of(context).maybePop();
+        Navigator.of(context).maybePop().then((bool didPop) {
+          if (!didPop)
+            SystemNavigator.pop();
+        }).catchError((dynamic error) {});
```

Please never catch and ignore errors.